src/bpmncwpverify/builder/process_builder.py

```diff
@@ -127,12 +127,9 @@ def build(self) -> Result[Process, BpmnStructureError]:
         try:
             self._bpmn.processes[self._process.id] = self._process
             self._construct_flow_network()
-            from bpmncwpverify.visitors.process_connectivity_visitor import (
-                ProcessConnectivityVisitor,
-            )
+            from bpmncwpverify.visitors.bpmnchecks.bpmnvalidate import validate_process
```

Comment:
Shouldn't all the imports be at the top of the file?  At least in Rust, that is expected for good style. We should likely do the same?

Reply:
In this case, I have it hear in order to avoid circular imports @ericmercer. I think that the case below makes sense to move them to the top though.

Reply:
Actually, you are right, I had it here originally for that reason but now after refactoring it is no longer necessary. Thank you! @ericmercer